docs/cvars.md

```diff
@@ -217,6 +217,7 @@
 |sar_hud_font_color|255 255 255 255|RGBA font color of HUD.<br>|
 |sar_hud_font_index|0|Font index of HUD.<br>|
 |sar_hud_frame|0|Draws current frame count.<br>|
+|sar_hud_fps|0|Shows the client's fps (frames per second) on the SAR hud.<br>1 - Show fps<br>2 - Show fps with fps cap<br>|
```

Comment:
you clearly didn't generate this from the script cuz this isn't what the hud element's help text is. it's fine to add to this file manually, but please make the texts match

Reply:
didn't know there was a script, changed this now anyways